lib/unobtainium/drivers/selenium.rb

```diff
@@ -89,6 +89,19 @@ def create(label, options)
           # :nocov:
         end
 
+        def construct_desired_caps_for_testdroid(options)
+          caps = options[:caps] || {}
```

Comment:
Already in this line, you could write:
````ruby
caps = options.delete(:caps) || {}
````
and delete line 95, I think. 

Or even can be changed to a oneliner
````ruby
options.merge!(options.delete(:caps) || {})
````
or 
````ruby
options.merge!(caps) if caps = options.delete(:caps)
````

if you want.